xml/ay_networking.xml

```diff
@@ -19,51 +19,73 @@
  </info>
 
    <para>
-    Network configuration is used to connect a single workstation to an
-    Ethernet-based LAN or to configure a dial-up connection. More complex
-    configurations (multiple network cards, routing, etc.) are also
-    provided.
+    Network configuration is mainly used to connect a single workstation to an
+    Ethernet-based LAN and it is commonly configured even before starting &ay;
+    in order to fetch the profile from a network location.
    </para>
 
    <para>
-    If the following setting is set to <literal>true</literal>, &yast;
-    will keep network settings created during the installation (via
-    <command>linuxrc</command>) and/or merge it with network settings from the
-    &ay; control file (if defined).
+    This network configuration is usually done through <command>linuxrc</command>
    </para>
+
    <note>
     <title>The <command>linuxrc</command> program</title>
     <para>
      For a detailed description of how <command>linuxrc</command> works and its
      keywords, see <xref linkend="appendix-linuxrc"/>.
     </para>
    </note>
+
+   <para>
+    By default &yast; keeps the network settings created during the installation.
+    Thus, linuxrc configured network could be considered the minimal network
```

Comment:
What about this?

*By default, &ay; copies the network settings that were used during the installation to the final system. This configuration is merged with the one that is defined in the &ay; profile. ...*

And, after that explanation, you can mention that you can skip copying the installation settings through the `keep_install_network` element.